bpf/init.sh

```diff
@@ -518,8 +518,9 @@ fi
 if [ "$MODE" = "vxlan" -o "$MODE" = "geneve" ]; then
 	ENCAP_DEV="cilium_${MODE}"
 	ip link show $ENCAP_DEV || {
-		ip link add name $ENCAP_DEV address $(rnd_mac_addr) mtu $MTU type $MODE external || encap_fail
+		ip link add name $ENCAP_DEV address $(rnd_mac_addr) type $MODE external || encap_fail
 	}
+	ip link set $ENCAP_DEV mtu $MTU
```

Comment:
Does this need to use `encap_fail`?
```suggestion
	ip link set $ENCAP_DEV mtu $MTU || encap_fail
```

Reply:
That will be more secure, I have updated the patch.